.github/workflows/isort.yml

```diff
@@ -0,0 +1,45 @@
+name: Code Formatter
+
+# Add `pull_request:` to apply on active PRs 
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  isort:
+    name: Run Isort
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Determine Branch Name
+        run: |
+          if [ "${{ github.event_name }}" = "pull_request" ]; then
+            echo "BRANCH_NAME=${{ github.head_ref }}" >> $GITHUB_ENV
+          else
+            BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/})
+            echo "BRANCH_NAME=$BRANCH_NAME" >> $GITHUB_ENV
+          fi
```

Comment:
The code for determining the branch name was only necessary if the workflow were to run on active PRs. This will not break the code but is no longer necessary. However, if we were to switch to have the workflow run on PRs then integrating that change will be very easy if we keep this code (addressed by the comment). Otherwise, it will take some work to change that behavior. Let me know if you think we should keep this code or not.